src/Ot2Rec/main.py

```diff
@@ -952,11 +1270,70 @@ def create_recon_yaml_stacked():
     Subroutine to create new yaml file for IMOD reconstruction
```

Comment:
Slightly confused here, does this deprecate the IMOD reconstruction since this runs Savu? Maybe need to change the function name to `create_savurecon_yaml_stacked`?

Reply:
I guess it might be easier to change the docstring instead, everything else seems to be correct